src/application/router/routes.ts

```diff
@@ -6,6 +6,7 @@ import NoteSettings from '@/presentation/pages/NoteSettings.vue';
 import ErrorPage from '@/presentation/pages/Error.vue';
 import Marketplace from '@/presentation/pages/Marketplace.vue';
 import type { RouteRecordRaw } from 'vue-router';
+import AddTool from '@/presentation/pages/AddTool.vue';
```

Comment:
lets move all marketplace pages under "marketplace" folder
```suggestion
import AddTool from '@/presentation/pages/marketplace/AddTool.vue';
```